drivers/javascript/util.coffee

```diff
@@ -101,10 +101,10 @@ convertPseudotype = (obj, opts) ->
             obj
 
 recursivelyConvertPseudotype = (obj, opts) ->
-    if obj instanceof Array
+    if Array.isArray obj
         for value, i in obj
             obj[i] = recursivelyConvertPseudotype(value, opts)
-    else if obj instanceof Object
+    else if obj and typeof obj is 'object'
```

Comment:
`obj and typeof obj is 'object'` now has two conditions in order to not be fooled by `null`, the `typeof` of which is `"object"`.